test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,140 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import logging
+import sys
+
+import pytest
+
+sys.path.insert(0, sys.path[0] + "/test/cql-pytest")
+import nodetool
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables, TextType, Column
+from test.pylib.util import unique_name
+
+logger = logging.getLogger(__name__)
+CONFIG = {"endpoint_snitch": "GossipingPropertyFileSnitch"}
+cluster_config = [
+    ([1, 2], 1),
+    ([1, 1, 2, 2], 2)
+]
+
+
+# Checks a cluster boot/operations in multi-dc environment with 30 nodes
+@pytest.mark.asyncio
+@pytest.mark.skip(reason="Too heavy for CI, but OK for local run")
+async def test_multidc(request: pytest.FixtureRequest, manager: ManagerClient) -> None:
+    logger.info("Creating a new cluster")
+    for i in range(30):
+        s_info = await manager.server_add(
+            config=CONFIG,
+            property_file={'dc': f'dc{i}', 'rack': 'myrack1'}
+        )
+        logger.info(s_info)
+    random_tables = RandomTables(request.node.name, manager, unique_name(), 3)
+    logger.info("Creating new tables")
+    await random_tables.add_tables(ntables=3, ncolumns=3)
+    await random_tables.verify_schema()
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("nodes_list, rf", cluster_config)
+async def test_putget_2dc_with_rf(
+        request: pytest.FixtureRequest, manager: ManagerClient, nodes_list: list[int], rf: int
+) -> None:
+    """Simple put-get test for 2 DC with one node each"""
```

Comment:
This comment is no longer accurate - it's just one of the two cluster_config options.